File: tests/fixtures-php/control-flow.php
<?php
function classify(int $n): int {
    if ($n > 0) {
        return 1;
    } elseif ($n < 0) {
        return -1;
    } else {
        return 0;
    }
}

echo classify(42);
echo classify(-7);
echo classify(0);

$total = 0;
for ($i = 0; $i < 10; $i++) {
    $total = $total + $i;
}
echo $total;

$count = 0;
$n = 1;
while ($n < 100) {
    $n = $n * 2;
    $count = $count + 1;
}
echo $count;
echo $n;

File: tests/fixtures-py/control-flow.py
def classify(n: int) -> int:
    if n > 0:
        return 1
    elif n < 0:
        return -1
    else:
        return 0

print(classify(42))
print(classify(-7))
print(classify(0))

total: int = 0
for i in range(10):
    total = total + i
print(total)

count: int = 0
n: int = 1
while n < 100:
    n = n * 2
    count = count + 1
print(count)
print(n)
